### ISSUE TITLE
Make `WPS113` respect `i_control_code` setting

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#951]
Currently `WPS113` raises a violation on

```python
# __init__.py
from classes.typeclass import typeclass as typeclass
```

Note this part: `typeclass as typeclass`
Why do we use it? Because we use `mypy` with `implicit_reexport = False` setting on.
So, when we define public interface for a module we need to use explicit reexport in a form of `from classes.typeclass import typeclass as typeclass`

So, that's why we need to tweak `WPS113` not to highlight these errors when we make libraries. 
In other words, when `i_control_code` is set to `True`, we need to ignore `WPS113` at all.

**Comment by ko3luhbka** [comment]
Hi, can I take this issue?

**Comment by sobolevn** [comment]
Done! 👍
